### ISSUE TITLE
Umbrella header missing `CALayer-KIFAdditions.h`

### ISSUE THREAD

**Issue by brennantaylor** [kif-framework/KIF#829]
It was just added as part of the last merge but not ending up in the umbrella header which is now causing an issue.

`<module-includes>:1:1: error: umbrella header for module 'KIF' does not include header 'CALayer-KIFAdditions.h' [-Werror,-Wincomplete-umbrella]`

**Comment by brennantaylor** [comment]
Sorry I would take a more active role in just opening the PR to address the issue but I'm trying to just get things working to unblock our dev team from updating to 7.3

**Comment by phatmann** [comment]
@RoyalPineapple: is this another thing we need to fix?

**Comment by RoyalPineapple** [comment]
@phatmann looks like it i'll open yet another PR

**Comment by RoyalPineapple** [comment]
I just YOLO merged it in
